auth0/data_source_auth0_user.go

```diff
@@ -0,0 +1,238 @@
+package auth0
+
+import (
+	"errors"
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/structure"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"gopkg.in/auth0.v4/management"
```

Comment:
@zombispormedio Thank you so much for your contribution. 

Could you modify the PR to rebase on current master & to use `auth0.v5` instead of `auth0.v4`?

Reply:
Done. I updated the branch and acc tests results in the PR description after re-run the tests. 
Thank you for your review.